### DISCUSSION TITLE
Previous CoApp versions

### DISCUSSION THREAD

**Discussion by TBill0919** [aclap-dev/video-downloadhelper#1102] [Help and Q&A]
### Your question

On MacosX 10.12.6 unable to update MacOS

VDH was working prior to update. Don't know what version I was using before update but it was prior to 04/29/23; date of directory created by VDH upon initial use. 
Version shown below (8.0.0.6) has been recently installed upon recommendation from here: https://github.com/aclap-dev/video-downloadhelper/discussions/288#discussioncomment-8966571
VDH and CoApp (vers 1.6.3) install without issue however CoApp isn't recognized by VDH. 
**Do you have to un-install prior versions before installing different version?**
**Do you have to close the browser prior to installing DVH or the CoApp on MacOS?**
Trying to nail down prior version that worked correctly.
Any advice would be appreciated. Thanks

### Addon & CoApp version and details

Video DownloadHelper
Version 8.0.0.6
Browser locale: en-US
Production build
Built on Tue Aug 22 2023 18:17:53 GMT+0200 (Central European Summer Time)
Build options: browser=firefox

Platform Mac x86-64
Browser Mozilla Firefox 115.11.0

**Comment by paulrouget** [comment]
> Version shown below (8.0.0.6) has been recently installed
> VDH and CoApp (vers 1.6.3) install without issue

This should work. But it's all very old code. Not sure how I can help though, it's very old code, and we do not support such old versions of VDH and MacOS.

**Reply by TBill0919** [reply]
Thanks for your reply, Paul. Just a head's up in the event others have the same issue.
After correct installation, within the "Settings" window it would display under "Companion app"; Unexpected error. **I reinstalled CoApp (vers 1.6.3) using the "dmg" version instead of the "pkg" version that I've been using.** The CoApp is now recognized (see attached pic) and DVH works as it did prior the updating. I've also turned off the "auto-update" feature. 
Again, thanks very much for your reply which confirmed the versions of the app and coapp do work and led me to believe it was something I was doing incorrectly.
I hope this can help others with the same issue.
-TBill
![DVH_Settings](https://github.com/user-attachments/assets/81ed40dd-f0cf-400f-8453-9800d7de81fa)
